### ISSUE TITLE
Page title = Last segment of the path

### ISSUE THREAD

**Issue by charlx** [gildas-lormeau/SingleFile#131]
Hi Gildas, I have tested other extensions but found your SingleFile as the best one so far. 
But I have a problem.

I have hundreds of html files saved on my PC with those "annoying_files" folders.
But I want to re-save them as single file with your extension.
Whenever I open those html files, the page title on the tab shows name different from my saved files.

Can you please add :  "**_%FILE% - the file in the last segment of the path without the final file extension_**" this feature like the "**Save Page WE**" extension ?

It will save me tons of hard works.
Thank you Genius.

![save page we saved filename options](https://user-images.githubusercontent.com/46587193/51014120-3dee1d80-1590-11e9-8d41-f8c155c8b2cc.jpg)

**Comment by charlx** [comment]
Yeah but unfortunately it's not working with **Bengali** language.

It is converting a html file with name **বাংলা ভাষা.html**  into this : **__E0_A6_AC_E0_A6_BE_E0_A6_82_E0_A6_B2_E0_A6_BE_20_E0_A6_AD_E0_A6_BE_E0_A6_B7_E0_A6_BE.htm.html**

Can you please make it work with Bengali Language like Save Page WE extension ?
:)

![it s converting bengali language into name below singlefile](https://user-images.githubusercontent.com/46587193/51053555-2d778a80-1604-11e9-9dc2-bf7f51b82f3a.jpg)

**Comment by gildas-lormeau** [comment]
I just published a new version which should fix this issue.

**Comment by charlx** [comment]
Yeah thanks a lot now Bengali language is working fine.
But there are still 2 old problems : 

1. A Dash/underscore symbol is added in the beginning. 
2. Extension from file path is not removing.

I just can use a renamer tool to remove dash and extra .html from all the files later.
But in case you want your tool to be flawless :)

![there are still 2 problems - singlefile](https://user-images.githubusercontent.com/46587193/51069070-3131fe00-1652-11e9-9368-4e628915f182.jpg)
